Userland/Libraries/LibLine/SuggestionManager.h

```diff
@@ -31,7 +30,7 @@ struct CompletionSuggestion {
     }
 
     CompletionSuggestion(DeprecatedString const& completion, ForSearchTag)
-        : text_string(completion)
+        : text(MUST(String::from_deprecated_string(completion)))
```

Comment:
Maybe take a `StringView` instead, there's no point in taking a `DeprecatedString` anymore as we're definitely allocating it again here.